sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_uamqp_transport.py

```diff
@@ -544,19 +590,24 @@ def get_updated_token(mgmt_auth):
             """
             Return updated auth token.
             :param mgmt_auth: Auth.
+            :type mgmt_auth: ~uamqp.authentication.JWTTokenAuth
+            :return: Updated auth token.
+            :rtype: str
             """
             return mgmt_auth.token
 
         @staticmethod
         def mgmt_client_request(mgmt_client, mgmt_msg, **kwargs):
             """
             Send mgmt request.
-            :param AMQP Client mgmt_client: Client to send request with.
+            :param uamqp.AMQPClient mgmt_client: Client to send request with.
             :param str mgmt_msg: Message.
             :keyword bytes operation: Operation.
             :keyword operation_type: Op type.
             :keyword status_code_field: mgmt status code.
             :keyword description_fields: mgmt status desc.
+            :return: Status code, description, and response.
+            :rtype: tuple[int, str, any]
```

Comment:
I think status code should be str? B/c we cast it to an int at the sdk level
```suggestion
            :rtype: tuple[str, str, uamqp.Message]
```